examples/ae_examples/cvae_dim_example/client.py

```diff
@@ -0,0 +1,66 @@
+import argparse
+from pathlib import Path
+from typing import Sequence, Tuple
+
+import flwr as fl
+import torch
+import torch.nn as nn
+import torchvision.transforms as transforms
+from flwr.common.typing import Config
+from torch.nn.modules.loss import _Loss
+from torch.optim import Optimizer
+from torch.utils.data import DataLoader
+
+from examples.cvae_dim_example.mnist_model import MnistNet
+from fl4health.clients.basic_client import BasicClient
+from fl4health.preprocessing.autoencoders.vae_dim_reduction import ClientConditionedProcessor
+from fl4health.utils.load_data import load_mnist_data
+from fl4health.utils.metrics import Accuracy, Metric
+from fl4health.utils.random import set_all_random_seeds
+from fl4health.utils.sampler import DirichletLabelBasedSampler
+
+
+class CVAEDimClient(BasicClient):
```

Comment:
The style convention for class names with acronyms is a bit ambiguous, but we've settle on acronyms having their first letter capitalized and the remainder lower-case (see `MnistNet` above). So this name would become `CvaeDimClient`